client/unit-tests/symbolic/collections/test_structures.py

```diff
@@ -245,7 +245,7 @@ def test_list():
     }
 
     # test decode from json dict
-    assert List[Float].decode_value([0.1, 0.2, 0.3]).get_value() == [  # type: ignore
```

Comment:
same comment about adding ticket to GitHub

Reply:
https://github.com/Striveworks/valor/issues/604